qtglean/metrics.yaml

```diff
@@ -38,3 +38,53 @@ performance:
     notification_emails:
       - mcleinman@mozilla.com
     expires: never
+
+session:
+  session_start:
+    type: datetime
+    description: |
+      The time the user starts a VPN session from the app
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-4064
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/5702#issuecomment-1397761161
+    send_in_pings:
+      - vpnsession
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - mcleinman@mozilla.com
+    expires: never
+
+  session_end:
+    type: datetime
+    description: |
+      The time the user ends a VPN session from the app
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/VPN-4064
+    data_reviews:
+      - https://github.com/mozilla-mobile/mozilla-vpn-client/pull/5702#issuecomment-1397761161
+    send_in_pings:
+      - vpnsession
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - mcleinman@mozilla.com
+    expires: never
```

Comment:
These two metrics are essentially meaningless when the sessions_started count is > 1. Please include this note in the description of both metrics. They should always be analysed together.